truffleruby/src/main/ruby/core/truffle/regexp_ops.rb

```diff
@@ -0,0 +1,15 @@
+module Truffle
+  class RegexpOps
```

Comment:
We usually avoid abbreviations, so I would name this `RegexpOperations` (similar to how we name these things in Java).